interfaces/mount/entry.go

```diff
@@ -141,3 +144,49 @@ func ParseEntry(s string) (Entry, error) {
 	e.CheckPassNumber = cpn
 	return e, nil
 }
+
+// LoadFSTab reads and parses an fstab-like file.
+//
+// The supported format is described by fstab(5).
+func LoadFSTab(reader io.Reader) ([]Entry, error) {
+	var entries []Entry
+	scanner := bufio.NewScanner(reader)
+	for scanner.Scan() {
+		s := strings.TrimSpace(scanner.Text())
+		if i := strings.Index(s, "#"); i != -1 {
+			s = strings.TrimSpace(s[0:i])
+		}
+		if s == "" {
+			continue
+		}
+		entry, err := ParseEntry(s)
+		if err != nil {
+			return nil, err
+		}
+		entries = append(entries, entry)
+	}
+	if err := scanner.Err(); err != nil {
+		return nil, err
+	}
+	return entries, nil
+}
+
+// SaveFSTab writes a list of entries to a fstab-like file.
+//
+// The supported format is described by fstab(5).
+// Note that there is no attempt to use atomic file write/rename tricks. The
+// created file will typically live in /run/snapd/ns/$SNAP_NAME.fstab and will
+// be done so, while holidng a flock-based-lock, by the snap-update-ns program.
+func SaveFSTab(writer io.Writer, entries []Entry) error {
+	var buf bytes.Buffer
+	for i := range entries {
+		if _, err := buf.WriteString(entries[i].String()); err != nil {
```

Comment:
This could be written as `if _, err := fmt.Fpritnf(buf, "%s\n", entries[i]); err != nil {` unless I'm mistaken, i.e. you can avoid the extra 2 lines for the buf.WriteRune() below.